html/pfappserver/root/static.alt/src/components/pfFormChosen.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <b-form-group horizontal :label-cols="(columnLabel) ? labelCols : 0" :label="$t(columnLabel)"
-    :state="isValid()" :invalid-feedback="getInvalidFeedback()" :class="['chosen-element', { 'mb-0': !columnLabel }, { 'is-focus': focus}]">
-    <b-input-group class="input-group-chosen">
+    :state="isValid()" :invalid-feedback="getInvalidFeedback()" 
+    class="chosen-element" :class="[{ 'mb-0': !columnLabel }, { 'is-focus': focus}]">
```

Comment:
You can simplify to a single object literal: `:class="{ 'mb-0': !columnLabel, 'is-focus': focus}"`